src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderTransformer.java

```diff
@@ -31,10 +29,8 @@ public class OffenderTransformer {
 
     private final Clock clock;
 
-    public InmateDetail transform(final Offender offender) {
-        return offender.getLatestBooking()
-            .map(this::transform)
-            .orElse(buildOffender(offender).build());
+    public InmateDetail transformWithoutBooking(final Offender offender) {
```

Comment:
this stops us calling `getLatestBooking`, which can only be called on the root offender